### PR TITLE
Peers: new metrics in 1.4.0

### PR DESCRIPTION
**Pull request by mDuo13**
Documents https://github.com/ripple/rippled/pull/3086/

### CODE REVIEW

**Review by ryangyoung — APPROVED** [review]
LGTM 👍

File: content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/peers.md [hunk 1]
```diff
@@ -39,114 +39,136 @@ An example of a successful response:
...
       {
-        "address": "98.167.120.212:51235",
-        "complete_ledgers": "18828845 - 18828973",
-        "latency": 99,
-        "ledger": "50A2577CE6EB8A92847C443BDA45F5C5F0A22B9C6F4B47DBA0C12BDA75001D01",
-        "load": 60,
-        "public_key": "n9LDBRoqPYY7RdkNXbX1dqZXVtUKcSqzs2CZPhTH7ymA9X7Xzmpj",
-        "uptime": 99625,
-        "version": "rippled-0.30.1-rc4"
+        "address": "[::ffff:94.237.49.50]:51235",
```
Comment (jwbusch) [inline content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/peers.md:153]: Would there be any benefit to masking these IPs?

**Review by jwbusch — APPROVED** [review]
LGTM, just had one question about IPs being displayed.

File: content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/peers.md (hunk 1 above)
Comment (mDuo13) [reply content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/peers.md:153]: I don't think it's necessary. These IPs are probably obtainable just by connecting any server to the peer-to-peer network and listening to the protocol messages.

**Pull request merged**